meson.build

```diff
@@ -58,4 +58,17 @@ if get_option('static')
     cflags += '-DLPM_STATIC'
 endif
 
-executable('lpm', lpm_source, dependencies: [zlib_dep, lzma_dep, mbedtls_dep, libgit2_dep, libzip_dep, lua_dep, microtar_dep], c_args: cflags)
+executable('lpm',
```

Comment:
also something worth mentioning
if we assign this executable to a variable (e.g. `lpm_exe`) it can be used in anything that consumes this project with no cost to lpm itself

Reply:
adam said this sounds useful